### ISSUE TITLE
Issue with Sonoff RF Bridge 

### ISSUE THREAD

**Issue by riyas1980** [AlexxIT/SonoffLAN#388]
Hello,

I am trying to use a 2Gang 433 RF switch with Sonoff Bridge. I have an issue. I have created 2 automations (One for button) like this :

![Button1](https://user-images.githubusercontent.com/35848952/107965764-2f626380-6fab-11eb-81bf-bb0e9200fa68.png)

![Button2](https://user-images.githubusercontent.com/35848952/107965823-4a34d800-6fab-11eb-8d4a-dfd6cf9efee8.png)

![Actions](https://user-images.githubusercontent.com/35848952/107965840-4e60f580-6fab-11eb-94d8-304074126920.png)

Actually, It works. But I can not switch on Button2 Directly. If I want to switch on  Button2, I need to press Button1 first before Button2. If I press directly Button2, it won't work. And to switch off (I am using switch.toggle function), I need to repeat the same step: Button1 1st then Button2.

Maybe I am doing something wrong in the trigger section. But I am not able to find it. Can you please me?

Thanks

**Comment by AlexxIT** [comment]
You shouldn't do automations on state. You should do automations on event. Have you read docs?

**Comment by riyas1980** [comment]
@AlexxIT I've read your readme file. But I am not that good in Yaml configuration. Following your suggestion, I've tried to do automations on event like this. 

![RF_Event](https://user-images.githubusercontent.com/35848952/107982013-8b38e680-6fc3-11eb-9a5d-58822d5057d6.png)

But now I have another issue.  As I told you before, I am using switch.toggle service to switch on or off (Please see the 3rd screenshot in my first message) . But even if I press the button only once, it switches on but it goes off immediately ( after 1 sec). What am I doing wrong here ?

Thansk

**Comment by riyas1980** [comment]
Actually it's working fine now. I have a Sonoff TX2. Before buying my bridge, it was paired with a 433 remote. Both remotes were conflicting. Once I deleted the pairing, everything is fine.

Sorry for the trouble and thanks again for your worderful work.
